src/rotation.js

```diff
@@ -2,7 +2,7 @@ import compose from "./compose";
 import {asin, atan2, cos, degrees, pi, radians, sin, tau} from "./math";
 
 function rotationIdentity(lambda, phi) {
-  return [lambda > pi ? lambda - tau : lambda < -pi ? lambda + tau : lambda, phi];
+  return [lambda > pi ? (lambda + pi) % tau - pi : lambda < -pi ? (lambda - pi) % tau + pi : lambda, phi];
```

Comment:
Could this be more simply

```js
return [lambda - Math.floor((lambda + pi) / tau) * tau, phi];
```

Reply:
I don't know… after 1e5 runs around the planet you start to see 1e-11 differences between those values.

I don't know which is closest to the "true" value, but more importantly, which will keep on giving the sine and cosine closest to the true values (some inconclusive tests at https://beta.observablehq.com/d/fb0735769648e8bb ).

So, whichever is fine for me at this stage. `floor` is defined in src/math.js.